### ISSUE TITLE
Can't get it working with angular 2.0

### ISSUE THREAD

**Issue by paulvanbladel** [frankwallis/plugin-typescript#17]
I'm combining this working ng-2 sample with the typescript plugin:
https://github.com/angular/ts-quickstart
(so this sample works when doing the typescript compilation manually)
I followed the instructions in the readme of plugin-typescript, but the compilation fails.
Any idea?

**Comment by frankwallis** [comment]
There were possibly a few things going on here - I have updated the plugin to use the latest version of the typescript compiler (1.5.0-beta). There is also a new compiler setting 'emitDecoratorMetadata' which needs to be set to true for constructor parameters to be automatically injected.

I have ported my [angular2 example](https://github.com/frankwallis/tower/tree/angular2) over from traceur to use plugin-typescript and that is now building, so take a look at that as an example. The latest published version of angular2 on npm does not support the typescript decorators, and it was not possible to install directly from github due to file extensions, so to import angular, zones, and reflect, it is just using the pre-bundled file which ships with the quickstart. There are some runtime errors which I need to look at, but it is building and starting up ok.
